Library/Homebrew/download_strategy.rb

```diff
@@ -389,6 +389,23 @@ def _fetch
   end
 end
 
+# Will convert "https://downloads.sourceforge.net/project/..." URLs to mirrorservice.org
+# In case environment variable HOMEBREW_NO_INSECURE_REDIRECT is set, so the Sourceforge is not ultimately broken
+class CurlSourceForgeSecureMirrorDownloadStrategy < CurlDownloadStrategy
+  def fetch
+    unless ENV["HOMEBREW_NO_INSECURE_REDIRECT"].nil?
```

Comment:
I would rather not modify processing of all sourceforge formulas always, adding mirror is meaningful only in this specific case, unless we introduce something like "HOMEBREW_DISABLE_INSECURE_DOWNLOADS" which would require all formulas to be provided through SSL enabled address.